pytorch_lightning/core/lightning.py

```diff
@@ -1465,6 +1465,18 @@ def tbptt_split_batch(self, batch, split_size):
 
         return splits
 
+    @property
+    def sharded_module(self) -> torch.nn.Module:
```

Comment:
Wouldn't it be better if this was grouped with the other properties?

Reply:
Yes it would, let me move it :)